### PR TITLE
Feature/flagging modal filters

### PR DESCRIPTION
**Pull request by mxhs**
## PR TITLE

**Description:** Sorry a lot here. Mostly structural/organization but also added a dispatch to dynamically populate flag chips with number of flags per discussion. Small bug with flagging a discussion but currently being worked on separately.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -22,3 +22,7 @@ Team A Members...
 
 1. Once you have cloned the repository, download the dependencies with `npm install`.
 2. Start the server with `npm start`.
+
+
+### Organisation
```
Comment (mattcanavan) [inline readme.md]: can you fix this typo on a future commit

**Review by mattcanavan — APPROVED** [review]
Looks good.

File: readme.md (hunk 1 above)
Comment (mxhs) [reply readme.md]: Good catch.  Fixed

**Review by bgmad — APPROVED** [review]
Should be good to go

**Pull request merged**